bld/queryDefaultNamelist.pl

```diff
@@ -34,7 +34,8 @@
 
 #-----------------------------------------------------------------------------------------------
 # Add $cfgdir to the list of paths that Perl searches for modules
-my @dirs = ( $cfgdir, "$cfgdir/../../../cime/utils/perl5lib" );
+my @dirs = ( $cfgdir, "$cfgdir/../../../cime/utils/perl5lib", 
```

Comment:
Is this going to first look in `$cfgdir/../../../cime/utils/perl5lib`, then look in `$cfgdir/../cime/utils/perl5lib`? If so, I'd change the order of these. My rationale is: If you're within a standalone checkout, there's a chance that there would be an unrelated cime directory at `$cfgdir/../../../cime/utils/perl5lib`.